src/applications/disability-benefits/686c-674/config/form.js

```diff
@@ -53,30 +52,28 @@ import {
   studentNetworthInformation,
 } from './chapters/674';
 
+const emptyMigration = savedData => savedData;
+const migrations = [emptyMigration];
+
 const formConfig = {
   urlPrefix: '/',
-  // NOTE: e2e tests will fail until the dependents_applications endpoint gets merged in to vets-api.
-  // All e2e tests will be disabled until then. If you need to run an e2e test, temporarily change
-  // dependents_appilcations to 21-686c.
-  submitUrl: `${environment.API_URL}/v0/dependents_applications`,
```

Comment:
submitUrl should be kept as is I believe

Reply:
Yeah, I made a pull that got botched. Fixed this.